articles/zcash-protocol/chapter_2.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: Notations
+Status: Done
+Level: "1"
+---
+# Chapter 2: Notations
+
+---
+
+- $\mathbb{B}$ : the type of bit values.
+- $\mathbb{B}^{\mathbb{Y}}$ : the type of byte values
+- $\mathbb{N}$ : the type of nonnegative integers.
+- $\mathbb{N}^{+}$ : the type of positive integers.
+- $\mathbb{Z}$ : the type of integers.
+- $\mathbb{Q}$ : the type of rational.
+- $x\ : \ T$ : $x$ has type $T$.
```

Comment:
$x\ \colon \ T$ means $x$ has type $T$.

Reply:
use "\colon" instead of ":"

Reply:
Create a table with two columns. First column is Denoted, second column is Meaning